server/models/db/workerTimeslot.js

```diff
@@ -21,7 +21,19 @@ async function updateWorkerAvailability(workerId, specificDate, newStatus) {
     return db.any(`update ${Tables.workerTimeslot} set status='${newStatus}' where date='${specificDate}' and worker_id='${workerId}';`);
 }
 
+async function checkWorkerAvailability(workerTimeslotId) {
+    //Selects all from workerTimeslot table 
+    const queryStatement = `select * from ${Tables.workerTimeslot}  where worker_timeslot_id = ${workerTimeslotId}`;
+    const queryOutput = await db.any(queryStatement);
+    //check if workertimeSlot is present 
+    if (_.isEmpty(queryOutput)) return false;
+    //check if available 
+    const currentStatus = _.includes(queryOutput, 'status'); 
+    if (_.includes(currentStatus, TimeslotStatus.available)) return true;
```

Comment:
Hi Nancy, 
For simplicity and better code coverage, I would recommend just combining lines 32 and 33 to `return _.includes(currentStatus, TimeslotStatus.available)`.

Reply:
Hi Amy,
Thank you for your suggestion!
My Action: I have simplified the code by following your recommendation of replacing lines 32 and 33 with `return _.includes(currentStatus, TimeslotStatus.available).`

You can see this change in line 32 of server/models/db/workerTimeslot.js

Please confirm if this takes care of your suggestion!